src/databricks/sql/auth/oauth_http_handler.py

```diff
@@ -0,0 +1,42 @@
+from http.server import BaseHTTPRequestHandler
```

Comment:
I expect some of this will be explained in the design doc: but why does `databricks-sql-connector` need to run an HTTP server? Is this for the browser to redirect a callback URL after the user successfully authenticates?

Reply:
correct. Interactive User to Machine OAuth code flow requires the oauth code to be returned on the redirect url where an http server is listening. This class provides the implementation which listens for oauth code on the configured redirect url (which should be http://localhsot:8020 or other ports if this one taken.)